src/components/Alert/Alert.stories.tsx

```diff
@@ -1,6 +1,9 @@
 import { Meta, Story } from '@storybook/react/types-6-0';
 
+import React from 'react';
```

Comment:
Since react 17, this import is not required to be present anymore.

Reply:
ok :)